### PR TITLE
[MIG] apps_product_creator: Migration to 12.0

### PR DESCRIPTION
**Pull request by martintorresvm**
depends on 
https://github.com/OCA/interface-github/pull/38
https://github.com/OCA/interface-github/pull/39

@Tecnativa

### CODE REVIEW

File: apps_product_creator/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+11.0.1.1.2 (2018-10-02)
```
Comment (pedrobaeza) [inline apps_product_creator/readme/HISTORY.rst:6]: You can add here `12.0.1.0.0`

**Review by pedrobaeza — APPROVED** [review]
Code review

File: apps_product_creator/data/product_attribute.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <field name="name">9.0</field>
+        <field name="attribute_id" ref="attribute_odoo_version"/>
+    </record>
+
+    <record id="odoo_version_100" model="product.attribute.value">
+        <field name="name">10.0</field>
+        <field name="attribute_id" ref="attribute_odoo_version"/>
+    </record>
+
+    <record id="odoo_version_110" model="product.attribute.value">
```
Comment (elicoidal) [inline apps_product_creator/data/product_attribute.xml:45]: add 12.0

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
Dependencies are merged, so you can activate this one (attending the comments)

**Comment by martintorresvm** [comment]
Changes done

**Comment by pedrobaeza** [comment]
Please fix tests and runbot

**Comment by martintorresvm** [comment]
Changes done

**Comment by martintorresvm** [comment]
Help with runbot is welcome @gurneyalex 

perhaps is this?
```
2019-04-12 15:38:11,329 163 INFO openerp_test odoo.addons.sms.wizard.send_sms: The `phonenumbers` Python module is not available. Phone number validation will be skipped. Try `pip3 install phonenumbers` to install it.
```

**Comment by pedrobaeza** [comment]
Look for warnings:
```
2019-04-12 15:38:12,948 163 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (odoo_module_id, odoo_module_version_id) of product.product() have the same label: Odoo Module.
```

**Comment by martintorresvm** [comment]
thanks @pedrobaeza changes done

**Comment by martintorresvm** [comment]
runbot and travis passed

**Pull request merged**